src/core/index.js

```diff
@@ -19,7 +19,8 @@ var core = module.exports = Object.assign(require('./const'), require('./math'),
     Container:              require('./display/Container'),
     Transform:              require('./display/Transform'),
     TransformStatic:        require('./display/TransformStatic'),
-    TransformManual:        require('./display/TransformManual'),
+    TransformBase:          require('./display/TransformBase'),
+    TransformManual:        require('./display/TransformBase'),
```

Comment:
I hear you, but it would be much better to add it to src/deprecation.js. This won't break pixi-spine and will actually give warnings for you to update pixi-spine. We can remove this specific deprecation with v4 release, but if someone forgets, it's not a big deal.